xmlfmt_test.go

```diff
@@ -1,10 +1,8 @@
-package xmlfmt_test
+package xmlfmt
```

Comment:
Please don't make unnecessary changes like this. It was coded like before for a reason.

Reply:
It has been restored to Package xmlfmt_test , Import "xmlfmt" local package and adds GO.MOD files